components/post_view/channel_intro_message/channel_intro_message.tsx

```diff
@@ -190,12 +202,12 @@ function createDMIntroMessage(channel: Channel, centeredIntro: string, teammate:
     );
 }
 
-function createOffTopicIntroMessage(channel: Channel, centeredIntro: string, stats: any, usersLimit: number, theme: any) {
+function createOffTopicIntroMessage(channel: Channel, centeredIntro: string, stats: any, usersLimit: number, theme: any, showAddMemberButton: boolean) {
     const isPrivate = channel.type === Constants.PRIVATE_CHANNEL;
     const children = createSetHeaderButton(channel);
     let totalUsers = 0;
-    if (stats && (typeof stats.TOTAL_USERS === 'number')) {
-        totalUsers = stats.TOTAL_USERS;
+    if (stats && (typeof stats.total_users_count === 'number')) {
+        totalUsers = stats.total_users_count;
```

Comment:
Is this type check needed given that the type defined for this is already `number` [here](https://github.com/mattermost/mattermost-redux/blob/3d1028034d7677adfda58e91b9a5dcaf1bc0ff99/src/types/users.ts#L118)?

If not, I believe we could simplify this to: 

```suggestion
   totalUsers = stats?.total_users_count | 0;
```

Also, just noted below that this is being also defaulted in the `index` file.

Reply:
ok!